### PR TITLE
Sniper aim

### PR DESCRIPTION
**Pull request by stevau5**
## What issue is this PR solving?
Closes: https://github.com/Luna-Interactive/catastrophe/issues/166

## Description of change
What is being added or modified with this pull request?
You can now zoom out while sniping in order to see more of the map!
If you let go of ADS, it will return to original view. 

## How to reproduce the changes
Please describe step by step how to reproduce your changes.
Start game, switch to sniper, and ADS. 

## Screenshots or GIFs
If applicable, show off your work!
ezgif com-video-to-gif (1)

### CODE REVIEW

**Comment by lyonelz96** [comment]
@stevau5  Siickkk, I like it! Just got many good jobs from the team, it's safe to say this is approoooveeeedddd!!!!!!!

File: Assets/Scripts/Player/PlayerInventory.cs [hunk 1]
```diff
@@ -66,6 +68,7 @@ void Start()
         inputState = GetComponent<Catastrophe.PlayerController>().inputState;
 
         selectWeapon(0);
+        currentlyHolding = "Assault";
```
Comment (yshrdbrn) [inline Assets/Scripts/Player/PlayerInventory.cs]: I think you can remove this line

Comment (stevau5) [reply Assets/Scripts/Player/PlayerInventory.cs]: Ou! Yes I forgot about this. Thanks!!

**Comment by stevau5** [comment]
Thanks @lyonelz96 and @yshrdbrn ! I took care of the line of code and changed it to AR.. Thanks for pointing it out.

**Pull request merged**